manifests/install.pp

```diff
@@ -143,6 +143,9 @@
 
   # Fetch and install the ShibCAS component.
   if $include_cas {
+    if $casclient_source == undef {
```

Comment:
Why not use validation to check this?

Reply:
I'd suggest the validation above and remove this if

Reply:
Isn't validate deprecated?

Reply:
I menat the new way to validate. With the Type in front of the parameter. See the next coment

Reply:
I need to ensure casclient_source isn't empty if include_cas is true.   Otherwise I don't care what the value is

Reply:
OK I understand now. I'm not sure there is a way to do that with the type validation.
Probably assert_type in an if